src/docfx/build/redirection/RedirectionMap.cs

```diff
@@ -32,15 +32,14 @@ public bool TryGetRedirectionUrl(string sourcePath, out string redirectionUrl)
             return false;
         }
 
-        public bool TryGetDocumentId(Document file, out (string id, string versionIndependentId) id)
+        public bool TryGetDocumentId(Document file, ref (string id, string versionIndependentId) id)
```

Comment:
use `out` to indicate it is an out parameter. `ref` are typically used to pass large structs by reference.